eth/catalyst/api.go

```diff
@@ -175,20 +182,17 @@ func (api *ConsensusAPI) makeEnv(parent *types.Block, header *types.Header) (*bl
 	return env, nil
 }
 
-func (api *ConsensusAPI) GetPayloadV1(payloadID hexutil.Bytes) (*ExecutableDataV1, error) {
-	hash := []byte(payloadID)
-	if len(hash) < 8 {
-		return nil, &InvalidPayloadID
-	}
-	id := binary.BigEndian.Uint64(hash[:8])
-	data, ok := api.preparedBlocks[id]
+func (api *ConsensusAPI) GetPayloadV1(payloadID PayloadID) (*ExecutableDataV1, error) {
+	log.Trace("GetPayload", "payloadID", payloadID)
```

Comment:
Nitpicks for @MariusVanDerWijden, could you rewrite these logs as

`log.Trace("Engine API request received", "method", "GetPayload", "id", payloadID)`

Specifically, it's cleaner if you have a log message that describes the situation (engine call) and the exact method is a contextual parameter. I fill it reads better, and also it avoids weird one-word log messages which look ugly :P.